src/games/kalambury/Kalambury.js

```diff
@@ -174,48 +144,41 @@ export const Kalambury = {
           G.startTime = new Date();
           G.canChangePhrase = true;
           SetNewPhrase(G, ctx);
-          G.remainingSeconds = 120;
+          G.turnEndTime = currentTime() + G.timePerTurn;
           LogAction(G, ctx, ctx.currentPlayer, "draw");
           ctx.events.setActivePlayers({ currentPlayer: "draw", others: "guess" });
         },
         onEnd: (G, ctx) => {
-          if (G.remainingSeconds <= 0) {
+          if (currentTime() >= G.turnEndTime) {
             LogAction(G, ctx, ctx.currentPlayer, "timeout", { previous: G.secret.phrase }, true);
             G.points[ctx.currentPlayer] -= 1;
           }
         },
-        endIf: (G, _ctx) => G.remainingSeconds <= 0,
+        endIf: (G, _ctx) => currentTime() >= G.turnEndTime,
         order: {
           first: () => 0,
           next: (G, ctx) => {
```

Comment:
It doesn't seem to work. I've created 2 people room, joined both slots, started the game. After the 2nd player guesses correctly the turn ends (points are added and phrase changes), but the 1st player is still drawing